client/js/lounge.js

```diff
@@ -1105,13 +1105,35 @@ $(function() {
 	chat.on("input", ".search", function() {
 		var value = $(this).val().toLowerCase();
 		var names = $(this).closest(".users").find(".names");
-		names.find(".user").each(function() {
-			var btn = $(this);
-			var name = btn.text().toLowerCase().replace(/[+%@~]/, "");
-			if (name.indexOf(value) > -1) {
-				btn.show();
+		// Find all the categories of users and loop through them.
+		names.find(".user-mode").each(function() {
```

Comment:
I have no idea why, but arrow functions don't seem to work in this case. I can't even type in the search box, but no errors are thrown anywhere. Not in the tests, not in the console. Maybe I am doing something wrong? I simply replaced `function()` with `() =>` and kept the rest the same.